src/main/java/frc/robot/Robot.java

```diff
@@ -27,10 +26,7 @@
    */
   @Override
   public void robotInit() {
-    // set default DriveTrainType preference
-    if (!Preferences.containsKey(Constants.DRIVE_TRAIN_TYPE)) {
-      Preferences.setString(Constants.DRIVE_TRAIN_TYPE, Constants.TANK);
-    }
+      Preferences.initString(Constants.DRIVE_TRAIN_TYPE, "DEFAULT");
```

Comment:
Make sure that this does not override the preference set by the dashboard. The docs say it does not put into the table if already exists. Can't tell the WPILib code is actually doing so. Need to test.

Reply:
My testing with the simulator indicates that it does what it is documented to do. Do you have some reason that it doesn't?